Assingment/Instagram/Instagram/SignUp/RegisterPasswordViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  RegisterPasswordViewController.swift
+//  Instagram
+//
+//  Created by limyunhwi on 2022/04/05.
+//
+
+import UIKit
+
+class RegisterPasswordViewController: UIViewController {
+    @IBOutlet weak var nextButton: UIButton!
+    @IBOutlet weak var passwordTextField: UITextField!
+    
+    var user: User?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        self.configureView()
+        self.configureInputField()
```

Comment:
```suggestion
        configureView()
        configureInputField()
```

self 키워드의 경우 반드시 써줘야 하는 경우 (클로저 내부 또는 명시적으로 클래스 내의 인스턴스를 말하고 싶을 때-클래스 내부의 변수명과 함수의 파라미터로 받은 이름이 같은 경우-.. 등 ..) 를 제외하고 사용을 지양하는 것이 코드의 간결함을 위해서 좋습니당 🥰